tendrl/gluster_integration/sds_sync/__init__.py

```diff
@@ -649,6 +649,9 @@ def _run(self):
                     _cluster.is_managed = "yes"
                     _cluster.save()
 
+                # check and enable volume profiling
+                self._enable_disable_volume_profiling()
```

Comment:
This function should be called only by the gluster-integration with tag "provisioner/$integration_id"